markdown/3.x/en/api/ff-record-list.api.md

```diff
@@ -7,6 +7,7 @@ ___
 | **records**&nbsp;(Array) | An Array of Objects containing the record data returned by FACT-Finder. |
 | **is-recommendation** (Boolean) | Indicates if this record-list is placed inside a ff-recommendation element. This attribute is set automatically by the ff-recommendation element and used for tracking purposes. |
 | **infinite-scrolling** (Boolean) | When present, the record list will load the next page when the bottom of the record list comes in view. |
+| **restore-scroll-position** (Boolean) (default: "true") | When present, the record list will restore the scroll position, which user was on before being redirected to record page. **NOTE** To work correctly, attribute `[data-redirect]` has to be present inside `ff-record`. For more information, see the [Tracking](documentation/3.x/tracking-guide).   |
```

Comment:
For Boolean default values the non-quoted values seem to be used throughout the documentation.
```suggestion
| **restore-scroll-position** (Boolean) (default: true) | When present, the record list will restore the scroll position, which the user was at before being redirected to a record page. **NOTE** To work correctly, attribute `[data-redirect]` has to be present inside `ff-record`. For more information, see the [Tracking](documentation/3.x/tracking-guide).   |
```